ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -580,6 +581,41 @@ def power(x1, x2, /):
     return ivy.pow(x1, x2)
 
 
+@with_supported_dtypes(
+    {"0.4.14 and below": ("float32", "float64", "int32", "int64")}, "jax"
```

Comment:
Can you give some reference(like the jax docs) where it mentions about these supported dtypes?

Reply:
@bipinKrishnan The reason I added this was due to some tests getting failed for paddle backend. I have now updated Jax frontend function to include all dtypes (i.e. removing supported dtypes handler) and added supported dtype handler in paddle backend. Hope this makes sense.